doc/nrf/applications.rst

```diff
@@ -0,0 +1,18 @@
+.. _applications:
+
+Applications
+############
+
+Applications consist of a fully integrated stack and interface drivers and libraries from the |NCS| and Zephyr.
```

Comment:
Applications typically include a fully integrated software stack, libraries and interface drivers from the |NCS| and its set of repositories.

Reply:
Not sure I understand ... Do you mean "they include a stack + libraries + interface drivers"?
I read the input as "they include a full stack, and they (resp. the stack) interface the following things: drivers and libraries".

Reply:
ah sorry, yes, I mean: "They include (i.e. in the build) stack + libraries + interface drivers"